starknet_py/tests/e2e/mock/compile_contracts.sh

```diff
@@ -1,32 +1,85 @@
 #!/bin/bash
-
+set -e
 MOCK_DIRECTORY=starknet_py/tests/e2e/mock
-CONTRACTS_DIRECTORY="$MOCK_DIRECTORY"/contracts
-CONTRACTS_COMPILED_DIRECTORY="$MOCK_DIRECTORY"/contracts_compiled
-
-# delete all artifacts except precompiled ones
-find $CONTRACTS_COMPILED_DIRECTORY -maxdepth 1 -type f -delete
+CONTRACTS_DIRECTORY_V1="$MOCK_DIRECTORY"/contracts_v1
```

Comment:
```suggestion
CONTRACTS_DIRECTORY_V1="$MOCK_DIRECTORY/contracts_v1"
```
Always double quote whole variables :+1: